metadata-ingestion/src/datahub/ingestion/source/dbt/dbt_common.py

```diff
@@ -688,12 +693,8 @@ def __init__(self, config: DBTCommonConfig, ctx: PipelineContext, platform: str)
                 self.config.owner_extraction_pattern
             )
         # Create and register the stateful ingestion use-case handler.
-        self.stale_entity_removal_handler = StaleEntityRemovalHandler(
-            source=self,
-            config=self.config,
-            state_type_class=GenericCheckpointState,
-            pipeline_name=self.ctx.pipeline_name,
-            run_id=self.ctx.run_id,
+        self.stale_entity_removal_handler = StaleEntityRemovalHandler.create(
```

Comment:
This gets used to call `self.stale_entity_removal_handler.add_urn_to_skip(node_datahub_urn)` at some point